dist/stat_data.py

```diff
@@ -247,26 +261,29 @@ def __init__(self, name, desc, flags=''):
     LogStat('log_scan_records', 'records processed by log scan'),
     LogStat('log_scan_rereads', 'log scan records requiring two reads'),
     LogStat('log_scans', 'log scan operations'),
-    LogStat('log_sync', 'log sync operations'),
-    LogStat('log_sync_dir', 'log sync_dir operations'),
-    LogStat('log_writes', 'log write operations'),
-    LogStat('log_write_lsn', 'log server thread advances write LSN'),
-
+    LogStat('log_slot_closes', 'consolidated slot closures'),
     LogStat('log_slot_coalesced', 'written slots coalesced'),
     LogStat('log_slot_consolidated', 'logging bytes consolidated'),
-    LogStat('log_slot_closes', 'consolidated slot closures'),
     LogStat('log_slot_joins', 'consolidated slot joins'),
     LogStat('log_slot_races', 'consolidated slot join races'),
     LogStat('log_slot_toobig', 'record size exceeded maximum'),
     LogStat('log_slot_toosmall',
         'failed to find a slot large enough for record'),
     LogStat('log_slot_transitions', 'consolidated slot join transitions'),
+    LogStat('log_sync', 'log sync operations'),
+    LogStat('log_sync_dir', 'log sync_dir operations'),
+    LogStat('log_write_lsn', 'log server thread advances write LSN'),
+    LogStat('log_writes', 'log write operations'),
 
     ##########################################
     # Reconciliation statistics
     ##########################################
     RecStat('rec_pages', 'page reconciliation calls'),
     RecStat('rec_pages_eviction', 'page reconciliation calls for eviction'),
+    RecStat('rec_pages_lookaside',
+        'page reconciliation block requires lookaside records'),
+    RecStat('rec_pages_restore',
+        'page reconciliation block requires in-memory restoration'),
```

Comment:
`rec_pages_lookaside` is the count of reconciled pages that required us to create any LAS records, the lookaside insert count is the count of LAS records we created, so it's 1-to-N.

`rec_pages_restore` is the count of reconciled pages where we did an update/restore cycle (that is, we didn't write the page, we found uncommitted records so we cleaned up the page and then re-instantiated it in memory). That's orthogonal to all of this work, but I noticed we weren't tracking it anywhere.